### ISSUE TITLE
Some banners are not displayed on Steam Wishlist

### ISSUE THREAD

**Issue by joshdigit** [jshackles/Enhanced_Steam#752]
Some banners are not displayed on Steam Wishlist (lots of DLC and some new games). Maybe it's because more than 700 names in my wishlist. But a few updates ago, everything worked perfectly.

**Comment by jshackles** [comment]
Yes this looks like the reason why.  I'm going to add some logic to the next update that should prevent Enhanced Steam from attempting to query the API so much (and fail, if it's over 200) which should clear up this issue.

**Comment by joshdigit** [comment]
Banners are not displayed again in Steam Wishlist after latest update (7.7). Can you fix it please?

**Comment by VanityFox** [comment]
So, Enhanced Steam is discontinued. There is now [Augmented Steam](https://github.com/tfedor/AugmentedSteam/). Perhaps your requested feature/bugfix is already implemented there! If not, you could move your issue there.
